grails-app/assets/javascripts/managementUnitReport.js

```diff
@@ -0,0 +1,36 @@
+function ManagementUnitReportSelectorViewModel() {
+    var self = this;
+    var now = convertToSimpleDate(new Date());
+    var d = new Date();
+    d.setFullYear(2018, 6, 1);
+    self.fromDate = ko.observable(d).extend({simpleDate:false});
+    self.toDate = ko.observable(now).extend({simpleDate:false});
+
+    self.muReportDownloadSummary = function() {
+        var summaryFlag = true;
+        generateMuReport(summaryFlag);
+    };
+
+    self.muReportDownload = function () {
+        generateMuReport();
+    };
+
+    function generateMuReport(summaryFlag = false) {
```

Comment:
I don't think default parameter values are supported in IE11 - this is a ES6 feature.